wallet-mobile-app-centbit/Components/HomeFourthCom.js

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react'
+import { View, Image, Text } from 'react-native'
+import { COLORS } from '../Constants/colors';
+import { HOMEIMAGSPOTIFY } from '../Constants/file';
+
+export default function HomeFourthComponent(){
+     return(
+          <View>
+               <View style={{
+                    backgroundColor: COLORS.colorblack,
```

Comment:
you need to stick with naming convection and colorblack *camelCase or snake_case